views/game.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+    <title>&#129304; &#128196; &#9986;</title>
+  </head>
+
+  <body>
+    <h1> <%= @player1 %>! Choose your weapon:</h1>
+
+    <form class="form-check" action="/weapon" method="post">
+      <div class="form-check">
+        <input class="form-check-input" type="radio" name="weapon" id="rock" value="Rock">
+        <label class="form-check-label" for="Rock">Rock</label>
+      </div>
+      <div class="form-check">
+        <input class="form-check-input" type="radio" name="weapon" id="paper" value="Paper">
+        <label class="form-check-label" for="Paper">Paper</label>
+      </div>
+      <div class="form-check">
+        <input class="form-check-input" type="radio" name="weapon" id="scissors" value="Scissors">
+        <label class="form-check-label" for="Scissors">Scissors</label>
+      </div>
+      <input type="submit" name="submit_weapon" value="Go!">
+    </form>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
My HTML is pretty rusty so I could be wrong here, but is it not normal practice to have `<script>` snippets in the `<head>` of an HTML doc, rather than the `<body>`?